BreakingChanges.txt

```diff
@@ -1,6 +1,7 @@
 Tracking Breaking Changes since 6.0
 
 - All: The dnx and net target frameworks have been unified, and DNXCore target framework has been renamed to Dotnet5.4. All DNX4.5.1 projects will take a dependency on the regular Windows Desktop dll.
+- Blobs: Change boundary checks for lease duration to be in line with those used in the REST endpoints
```

Comment:
We should use "Change boundary checks for lease duration and break lease period to be in line with those used in the REST endpoints." and we need to use this message as well in the changelog (noticed there was a typo in the changelog below).